imageroot/actions/import-module/40Migration

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e -o pipefail
+exec 1>&2 # Redirect any output to the journal (stderr)
+
+# Prepare an initialization script that restores the dump file
+mkdir -vp initdb.d
+mv -v roundcubemail.sql initdb.d
+#do the bash file to restore and exit once done
+cat - >initdb.d/roundcubemail_restore.sh <<'EOS'
+# Read dump file from standard input:
+cd docker-entrypoint-initdb.d
+mysql -uroundcubemail -proundcubemail roundcubemail < roundcubemail.sql
```

Comment:
The image entrypoint script already loads .sql dump files. Why does DB restore run here?
```suggestion
```

Reply:
good idea